src/Message/BatchMessage.php

```diff
@@ -100,26 +100,34 @@ public function finalize(): void
 
         if (empty($this->domain)) {
             throw new RuntimeException('You must call BatchMessage::setDomain before sending messages.');
-        } elseif (empty($message['from'])) {
```

Comment:
I removed all these elseif

Reply:
I'm assuming the reason is that there are other checks are in place before to set the values; however, what happens if we don't call those methods? Do you want to rely on the API instead of throwing an Exception and warn the developer?

Reply:
Oh, We still to all the checks. I just removed `elseif` and replaced them with `if`. It would be the same since the body of the `if` throws an exception.